app/Http/Controllers/V4DB/MangaController.php

```diff
@@ -730,9 +719,22 @@ public function reviews(Request $request, int $id)
             $results->isEmpty()
             || $this->isExpired($request, $results)
         ) {
-            $page = $request->get('page') ?? 1;
-            $manga = $this->jikan->getMangaReviews(new MangaReviewsRequest($id, $page));
-            $response = \json_decode($this->serializer->serialize($manga, 'json'), true);
+
+            $response = (new ScraperHandler(
+                'MangaReviews',
+                MangaReviewsRequest::class,
+                (new QueryResolver())
+                    ->setNewQuery(new Query('id', $id))
+                    ->setNewQuery(new Query('page', $request->get('page') ?? 1))
+                    ->setNewQuery(new Query(
+                        'sort',
+                        $request->get('sort') ?? Constants::REVIEWS_SORT_MOST_VOTED,
+                        new ValidationTypeEnum([Constants::REVIEWS_SORT_MOST_VOTED, Constants::REVIEWS_SORT_NEWEST, Constants::REVIEWS_SORT_OLDEST]),
+                        new BadRequestException('Invalid sort for reviews. Please refer to the documentation: https://docs.api.jikan.moe/')
+                    ))
+                    ->setNewQuery(new Query('spoilers', $request->get('spoilers') ?? false))
+                    ->setNewQuery(new Query('preliminary', $request->get('preliminary') ?? false))
+            ))->getSerializedJSON();
```

Comment:
I'll convert this into dep injection. But I disagree with the ideal outcome. We already have Jikan registered in app so calling it like this would not be required:
`$reviews = $this->scraperHandler->getReviews($id, $page, $sort, $spoilers, $preliminary);`

Laravel does not have a helper function defined for serialization here, we're using jms-serializer.